src/app/side-bar/list/list.component.html

```diff
@@ -0,0 +1,39 @@
+<app-list-header></app-list-header>
+<app-sub-header-list></app-sub-header-list>
+<div class="list-container ml-24">
+  <p-table #dt [value]="users">
+    <ng-template pTemplate="header">
+      <tr>
+        <th>
+          <p-tableHeaderCheckbox data-cy="checkAll"></p-tableHeaderCheckbox>
+        </th>
+        <th pSortableColumn="code">List Item<p-sortIcon field="code"></p-sortIcon></th>
+        <th pSortableColumn="name">Create on <p-sortIcon field="name"></p-sortIcon></th>
+        <th pSortableColumn="category">Actions <p-sortIcon field="category"></p-sortIcon></th>
+      </tr>
+    </ng-template>
+    <ng-template pTemplate="body" let-users>
+      <tr class="p-selectable-row">
+        <td>
+          <p-tableCheckbox [value]="users" data-cy="checkBox"></p-tableCheckbox>
+        </td>
+        <td fxLayoutAlign="start center">
+          {{users.list_item}}
+        </td>
+        <td>{{users.Create_on}}</td>
+        <td><div fxLayoutGap="20px">
+          <i class="pi pi-pencil"></i>
+          <i class="pi pi-trash"></i>
+          <i class="pi pi-ellipsis-h"></i>
+        </div></td>
+      </tr>
+    </ng-template>
+  </p-table>
+  <div fxLayout="row" fxLayoutGap="50%" class="list-container__paginator">
```

Comment:
`fxLayoutGap="50%" `??? please refactor it. try to use space-between

Reply:
Resolved...................Thank you